requirements.txt

```diff
@@ -0,0 +1,18 @@
+cython
+numpy
+scipy
+sklearn
+pandas
+tensorflow-datasets>=3.2.1,<4.0.0
```

Comment:
@fengredrum Please update `tensorflow-datasets` version

Reply:
And add `tensorflow-io` in the requirements (make sure this is the same with [setup.py](https://github.com/TensorSpeech/TensorFlowASR/blob/main/setup.py))

Reply:
@usimarit Done 😁